design/sketch.webidl

```diff
@@ -640,43 +641,43 @@ enum GPUStoreOp {
 };
 
 dictionary GPURenderPassColorAttachmentDescriptor {
-    GPUTextureView attachment;
-    GPUTextureView? resolveTarget;
+    required GPUTextureView attachment;
+    GPUTextureView? resolveTarget = null;
 
-    GPULoadOp loadOp;
-    GPUStoreOp storeOp;
-    GPUColor clearColor;
+    GPULoadOp loadOp = "clear";
+    GPUStoreOp storeOp = "store";
+    GPUColor clearColor = {r: 0.0, g: 0.0, b: 0.0, a: 1.0};
 };
 
 dictionary GPURenderPassDepthStencilAttachmentDescriptor {
-    GPUTextureView attachment;
+    required GPUTextureView attachment;
 
-    GPULoadOp depthLoadOp;
-    GPUStoreOp depthStoreOp;
-    float clearDepth;
+    GPULoadOp depthLoadOp = "clear";
+    GPUStoreOp depthStoreOp = "store";
+    float clearDepth = 1.0;
 
-    GPULoadOp stencilLoadOp;
-    GPUStoreOp stencilStoreOp;
-    u32 clearStencil;
+    GPULoadOp stencilLoadOp = "clear";
+    GPUStoreOp stencilStoreOp = "store";
+    u32 clearStencil = 0;
 };
 
 dictionary GPURenderPassDescriptor {
-    sequence<GPURenderPassColorAttachmentDescriptor> colorAttachments;
-    GPURenderPassDepthStencilAttachmentDescriptor? depthStencilAttachment;
+    required sequence<GPURenderPassColorAttachmentDescriptor> colorAttachments;
+    GPURenderPassDepthStencilAttachmentDescriptor? depthStencilAttachment = null;
```

Comment:
Our idl parser doesn't allow dictionary types to have a `?`. (I think it wants `GPURenderPassDepthStencilAttachmentDescriptor depthStencilAttachment = null` maybe.) But webidl2.js does allow it, so I'm not sure if this is indicative of a problem.